platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/DurationGauge.java

```diff
@@ -35,8 +38,8 @@ public interface DurationGauge extends Metric {
      * {@inheritDoc}
      */
     @Override
-    default MetricType getMetricType() {
-        return MetricType.GAUGE;
+    default String getMetricType() {
```

Comment:
Returning a `String` instead of an enum seems like a step back. Why is this necessary?

Reply:
because the basic metrics and the platform specific metrics are splitted now. Based one that an enum is not possible anymore